webapp/channels/src/plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -328,18 +328,16 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
                         bsRole='toggle'
                         dropdownOpen={this.state.dropdownOpen}
                     >
-                        <OverlayTrigger
-                            delayShow={Constants.OVERLAY_TIME_DELAY}
+                        <WithTooltip
+                            id='removeIcon'
                             placement='bottom'
-                            overlay={this.state.dropdownOpen ? <></> : (
-                                <Tooltip id='removeIcon'>
-                                    <div aria-hidden={true}>
-                                        <FormattedMessage
-                                            id='generic_icons.plugins'
-                                            defaultMessage='Plugins'
-                                        />
-                                    </div>
-                                </Tooltip>
+                            title={this.state.dropdownOpen ? <></> : (
+                                <div aria-hidden={true}>
+                                    <FormattedMessage
+                                        id='generic_icons.plugins'
+                                        defaultMessage='Plugins'
+                                    />
+                                </div>
```

Comment:
You can directly pass in the <FormattedMessage instead of the wrapper div here, since the WithTooltip will/should handle that for you